src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -6,6 +6,7 @@ Release History
 0.1.19
 ++++++
 * webapp: fix a bug that downloaded log file might be invalid
+* webapp: fix --web-server-logging to accept storage logging
```

Comment:
We cut a release a yesterday, so please add your history update to 0.1.20. Please also bump the version in `setup.py` to 0.1.20 as well.